src/main/java/dev/mccue/resolve/graph/DependencyTreeInterface.java

```diff
@@ -0,0 +1,23 @@
+package dev.mccue.resolve.graph;
+
+import dev.mccue.resolve.core.Dependency;
+import dev.mccue.resolve.doc.Coursier;
+import dev.mccue.resolve.doc.Incomplete;
+
+import java.util.List;
+
+@Incomplete
+@Coursier("https://github.com/coursier/coursier/blob/f5f0870/modules/core/shared/src/main/scala/coursier/graph/DependencyTree.scala")
+public interface DependencyTreeInterface {
+    Dependency dependency();
+
+    boolean excluded();
+
+    String reconciledVersion();
+
+    String retainedVersion();
+
+    List<DependencyTreeInterface> children();
+}
+
```

Comment:
What is the purpose of this interface? I don't see it implemented/extended anywhere

Reply:
I'm not sure where this came from, I don't remember it from before but my only thinking is it's trying to be the abstract class from the scala version.

Reply:
Will delete it